### ISSUE TITLE
delete frames during full save. this only happens when the frames are of imported images

### ISSUE THREAD

**Issue by TizioProgrammantae** [opentoonz/opentoonz#3859]
delete frames during full save. this only happens when the frames are of imported images
![Immagine 2021-04-04 215528](https://user-images.githubusercontent.com/81040838/113520025-87235100-9590-11eb-8b33-7c1faf0156f5.jpg)

**Comment by RattusaurusRex** [comment]
Does the imported image say "Single Frame" in the Level Strip?  Did you add new frames to it from the Xsheet? If the image is named in a certain format (like image.0001.png) then you should be able to add new frames to the level inside of OT.  If it is a Single Frame image, any new frames you add will not be saved.
![image](https://user-images.githubusercontent.com/60277069/113520128-df813100-9545-11eb-8a52-2327f6339940.png)

**Comment by tizio573** [comment]
> Does the imported image say "Single Frame" in the Level Strip? Did you add new frames to it from the Xsheet? If the image is named in a certain format (like image.0001.png) then you should be able to add new frames to the level inside of OT. If it is a Single Frame image, any new frames you add will not be saved.
> ![image](https://user-images.githubusercontent.com/60277069/113520128-df813100-9545-11eb-8a52-2327f6339940.png)

nope, is a png image and it has only one frame, if I copy the image frame in OT and I mdify it, it will be deleted at the saving

**Comment by RodneyBaker** [comment]
Closing as likely resolved and/or abandoned.
@TizioProgrammantae please let us know if you are still expriencing the problem.
Thanks
